src/platform/testing/e2e/cypress/support/form-tester/index.js

```diff
@@ -490,6 +491,7 @@ const testForm = testConfig => {
     setup = () => {},
     setupPerTest = () => {},
     skip,
+    runAxeCheck = true,
```

Comment:
In the `beforeEach`, I would `cy.wrap(runAxeCheck).as('runAxeCheck')` and then as needed, do this:

```
cy.get('@runAxeCheck')
  .then(runAxeCheck => if (runAxeCheck) cy.axeCheck());
```